Makefile

```diff
@@ -40,16 +41,19 @@ $(seabios):
 	cp scripts/seabios.config seabios/.config
 	make -C seabios
 
+$(seabios_blob): $(seabios)
```

Comment:
Did you mean to check in `mythril/src/blob/bios.bin`? Wouldn't this overwrite that?

Reply:
Yep, that's my bad